src/Settings/Settings/SettingsGeneral.php

```diff
@@ -143,22 +153,41 @@ public function getSettingsGlobalData(): array
 				'checkboxesContent' => [
 					[
 						'component' => 'checkbox',
-						'checkboxName' => $this->getSettingsName(self::SETTINGS_GENERAL_DISABLE_DEFAULT_STYLES_KEY),
 						'checkboxId' => $this->getSettingsName(self::SETTINGS_GENERAL_DISABLE_DEFAULT_STYLES_KEY),
-						'checkboxLabel' => __('Disable default Styles', 'eightshift-forms'),
+						'checkboxLabel' => __('Disable default styles', 'eightshift-forms'),
 						'checkboxIsChecked' => $this->getOptionValue(self::SETTINGS_GENERAL_DISABLE_DEFAULT_STYLES_KEY) === 'true',
 						'checkboxValue' => 'true',
 					],
 					[
 						'component' => 'checkbox',
-						'checkboxName' => $this->getSettingsName(self::SETTINGS_GENERAL_DISABLE_DEFAULT_SCRIPTS_KEY),
 						'checkboxId' => $this->getSettingsName(self::SETTINGS_GENERAL_DISABLE_DEFAULT_SCRIPTS_KEY),
-						'checkboxLabel' => __('Disable default Scripts', 'eightshift-forms'),
+						'checkboxLabel' => __('Disable default scripts', 'eightshift-forms'),
 						'checkboxIsChecked' => $this->getOptionValue(self::SETTINGS_GENERAL_DISABLE_DEFAULT_SCRIPTS_KEY) === 'true',
 						'checkboxValue' => 'true',
 					]
 				]
 			],
+			[
+				'component' => 'checkboxes',
+				'checkboxesFieldLabel' => \__('Form behavior', 'eightshift-forms'),
+				'checkboxesFieldHelp' => \__('Set all form behavior options in one place', 'eightshift-forms'),
```

Comment:
This looks odd 🤔 In other places you didn't prefix the `__()` function, so you can omit it here as well.

Reply:
i have remove all \__ and replaced it with __ on the whole project